llvm/lib/SYCLLowerIR/ESIMD/LowerESIMD.cpp

```diff
@@ -1086,7 +1087,8 @@ static uint64_t getIndexFromExtract(ExtractElementInst *EEI) {
 /// of vector load. The parameter \p IsVectorCall tells what version of GenX
 /// intrinsic (scalar or vector) to use to lower the load from SPIRV global.
 static Instruction *generateGenXCall(Instruction *EEI, StringRef IntrinName,
-                                     bool IsVectorCall, uint64_t IndexValue) {
+                                     bool IsVectorCall, uint64_t IndexValue,
+                                     Type *UseType = nullptr) {
```

Comment:
nit: with the new changes 'UsedType' is always passed to this function.
```suggestion
                                     Type *UseType) {
```

Reply:
good catch, thx